.github/workflows/server-test.yaml

```diff
@@ -22,10 +22,11 @@ jobs:
           go mod download
           make generate
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3
         with:
           version: v1.46.2
           working-directory: server
+          skip-pkg-cache: true
```

Comment:
updated golangci-lint-action. 

As both v2 and v3 outputs so many "File exists" error, I added this line. ref: https://github.com/golangci/golangci-lint-action/issues/244